### ISSUE TITLE
example文件夹里的model.py有错误

### ISSUE THREAD

**Issue by N-Kingsley** [jiangxiaoshuaiya/BertHub#5]
第64行中的hidden_num找不到
for i in range(hidden_num):
      total_output.append(last_hidden_state[-(i+1)][:, 0])

第81行和140行的ModelOutput找不到
return ModelOutput(
            loss=loss,
            logits=reshaped_logits,
            hidden_states=outputs.hidden_states,
            attentions=outputs.attentions,
        )

**Comment by jiangxiaoshuaiya** [comment]
> 第64行中的hidden_num找不到
> for i in range(hidden_num):
> total_output.append(last_hidden_state[-(i+1)][:, 0])
> 
> 第81行和140行的ModelOutput找不到
> return ModelOutput(
> loss=loss,
> logits=reshaped_logits,
> hidden_states=outputs.hidden_states,
> attentions=outputs.attentions,
> )

这个模型还未完善
